.github/workflows/golangci-lint.yml

```diff
@@ -26,24 +26,24 @@ jobs:
            GOOS: ${{ matrix.GOOS }}
         run: echo Go GOOS=$GOOS
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
What does this fix?

Reply:
Ah just a general version bump? https://github.com/golangci/golangci-lint-action/releases

Reply:
Yes, I was fighting with an issue that only happened on CI, so I ended up updating golangci-lint-action action and its "dependencies".

At the end it was all my fault because my PR didn't have the latest commit from `main`, hence I didn't get any lint errors locally.

Reply:
Let's keep the update anyway